Orange/widgets/data/owtranspose.py

```diff
@@ -34,6 +34,9 @@ class Outputs:
     feature_names_column = ContextSetting(None)
     auto_apply = Setting(True)
 
+    class Warning(OWWidget.Warning):
+        duplicate_names = Msg("Duplicate feature names are indexed.")
```

Comment:
I'm not sure that "to be indexed" is a proper phrase here. "Duplicate names have been given indices" could be better, but too long. What about "Values of '{}' are not unique.\nTo avoid multiple features with the same name, names were augmented with indices."? The second line only shows as tooltip.

Reply:
The second option is not much shorter once you add the name(s).

Reply:
I prefer the first suggestion.

Reply:
What about without the variable name? And then mentioning the variable name in the second line?